BUILDING.md

```diff
@@ -115,6 +115,7 @@ make generate
 >   * `no_btrfs`: A build tag disables building the Btrfs snapshot driver.
 >   * `no_devmapper`: A build tag disables building the device mapper snapshot driver.
 >   * `no_zfs`: A build tag disables building the ZFS snapshot driver.
+> * `containerd_no_dynamic_plugins`: A build tag disables dynamic plugins.
```

Comment:
nit: seems this line needs some indent to align with other lines. Also given all other tag is in `no_xxx` format, should we just call it `no_dynamic_plugins` or `no_containerd_dynamic_plugins`?

Reply:
> seems this line needs some indent to align with other lines.

Actually if you look above you'll see that it's the correct alignement, immediately above is a list specifically around snapshots.

> given all other tag is in no_xxx format, should we just call it no_dynamic_plugins or no_containerd_dynamic_plugins?

Good catch, I agree. Both of your proposals work for me honestly, the others don't mention `containerd` so maybe `no_dynamic_plugins` is more idiomatic (and unlike `static_build` I think it's explicit enough to not have an unexpected impact in other dependencies)

Reply:
> immediately above is a list specifically around snapshots.

Ah I see.

> `no_dynamic_plugins` is more idiomatic

SGTM